src/test/java/org/broadinstitute/hellbender/utils/dataflow/DataflowUtilsUnitTest.java

```diff
@@ -49,7 +50,7 @@ public void testThrowException() {
     @Test
     public void testReadFromFileFn(){
         List<SimpleInterval> intervals = Lists.newArrayList(new SimpleInterval("chr7:1-202"), new SimpleInterval("chr8:2-202"));
-        DoFn<File, Read> readfn = new DataflowUtils.LoadReadsFromFileFn(intervals);
+        DoFn<File, Read> readfn = new DataflowUtils.LoadReadsFromFileFn(intervals, ValidationStringency.SILENT);
```

Comment:
I'm going to assume you're asking @DR and not me. I think SILENT should be the default because it's what we're using as a default in many other places:
ReadsDataSource:123, and it's the default value in ReadsSource.getReadPCollection too.